geppetto.js/geppetto-ui/src/3d-canvas/Canvas.js

```diff
@@ -98,6 +100,25 @@ class Canvas extends Component {
     this.sceneRef.current.removeChild(
       this.threeDEngine.getRenderer().domElement
     );
+    this.sceneRef.current.removeEventListener('keydown', this.keyboardEventHandler)
+  }
+
+  keyboardEventHandler(event){
+    switch(event.code){
+      case 'ArrowRight':
```

Comment:
Maybe we can have the keys used configurable (I'm thinking that some people might prefer wasd instead of arrow keys) but I don't have a strong opinion on this. cc @ddelpiano @paularah